### PR TITLE
Myshor wanted this

### PR DESCRIPTION
**Pull request by Gusted**
For any info contact @Myshor

### CODE REVIEW

**Comment by Myshor** [comment]
i won't review the code... but functionality works ;)

**Comment by Gusted** [comment]
`Deep code`? Holmy Did I sign up for goverment jobs or Google?

**Comment by sprinkledcheese** [comment]
This fixes blobs in chrome?

**Comment by Gusted** [comment]
If you have Selector in IGNORE IMAGE ANALYZE it won't do anything with the URL. So e.g. If their is a blob created on a url you paste in the Selector into the section have a refresh poof no more blob.

**Comment by Myshor** [comment]
> 
> 
> This fixes blobs in chrome?

This adds possibility to exclude some classes for creating blobs for them. Will help me to refactor all blobs in facebook dynamic fixes. 😅

**Comment by sprinkledcheese** [comment]
Wow, this solves a lot of problems.

**Comment by Gusted** [comment]
:D I will try to push it true after #3120

**Comment by Gusted** [comment]
Btw, If someone wants to answer `specific user case`
 @sprinkledcheese proves you wrong :P

File: src/inject/dynamic-theme/modify-css.ts [hunk 1]
```diff
@@ -233,7 +233,20 @@ const gradientRegex = /[\-a-z]+gradient\(([^\(\)]*(\(([^\(\)]*(\(.*?\)))*[^\(\)]
...
 const awaitingForImageLoading = new Map<string, ((imageDetails: ImageDetails) => void)[]>();
 
-function getBgImageModifier(prop: string, value: string, rule: CSSStyleRule, isCancelled: () => boolean): string | CSSValueModifier {
+function shouldIgnoreImage(element: CSSStyleRule, selectors: string[]) {
+    if (!element) {
+        return false;
+    }
+    for (let i = 0; i < selectors.length; i++) {
+        const ingnoredSelector = selectors[i];
+        if (element.selectorText.match(ingnoredSelector)) {
```
Comment (alexanderby) [inline src/inject/dynamic-theme/modify-css.ts:242]: `String.match` returns a match object, so the condition will be true even for the part of the selector (e.g. 'div.beautiful' will be true for `div` or `.header .bg` will be true for .`header`). So I think it should be strict `===` comparison.

File: src/inject/dynamic-theme/inline-style.ts [hunk 2]
```diff
@@ -224,7 +224,7 @@ export function overrideInlineStyle(element: HTMLElement, theme: FilterConfig, i
     function setCustomProp(targetCSSProp: string, modifierCSSProp: string, cssVal: string) {
         const {customProp, dataAttr} = overrides[targetCSSProp];
 
-        const mod = getModifiableCSSDeclaration(modifierCSSProp, cssVal, null, null);
+        const mod = getModifiableCSSDeclaration(modifierCSSProp, cssVal, null, ignoreImageSelectors, null,);
```
Comment (alexanderby) [inline src/inject/dynamic-theme/inline-style.ts:227]: Extra comma after `null`

**Pull request merged**